src/core/shaderlib/project/viewport-uniforms.js

```diff
@@ -35,13 +36,14 @@ const VECTOR_TO_POINT_MATRIX = [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0];
 const IDENTITY_MATRIX = [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1];
 const DEFAULT_PIXELS_PER_UNIT2 = [0, 0, 0];
 
+const memoized = memoize(_getUniformsFromViewport);
```

Comment:
Nit: Just calling a function `memoized` seems confusing. Rather than an adjective, better stick with verb-name convention. `memoizedGetCoreUniforms` maybe?